### DISCUSSION TITLE
Feature request: screenshot support

### DISCUSSION THREAD

**Discussion by KreshnikUkmata** [spatie/laravel-support-bubble#22] [Ideas]
Can we add the ability to add screenshots in the support bubble, that would be really useful with error reporting, thanks in advance

**Comment by freekmurze** [comment]
Unless it's really simple to do, adding a screenshot is out of scope for this package.

**Reply by michellaurent** [reply]
Maybe a simple way to add attachment(s) to the message would already be enough

**Reply by freekmurze** [reply]
We currently have no plans for upload functionalities. 

If the code to achieve is simple, we'd accept a PR with tests.

**Reply by michellaurent** [reply]
Maybe a good one for the Hacktoberfest contest ;-)

**Reply by freekmurze** [reply]
You might win our hacktoberfest contest if it is a quality PR
https://twitter.com/spatie_be/status/1443577622810841088
